clang/lib/Parse/ParseTemplate.cpp

```diff
@@ -316,7 +318,8 @@ Parser::ParseConceptDefinition(const ParsedTemplateInfo &TemplateInfo,
   const IdentifierInfo *Id = Result.Identifier;
   SourceLocation IdLoc = Result.getBeginLoc();
 
-  DiagnoseAndSkipCXX11Attributes();
+  ParsedAttributes Attrs(AttrFactory);
+  MaybeParseAttributes(PAKM_GNU | PAKM_Declspec | PAKM_CXX11, Attrs);
```

Comment:
Did you check MSVC for the `PAKM_Declspec` ?

Reply:
I checked that `deprecated` is supported by `__declspec` syntax https://learn.microsoft.com/en-us/cpp/cpp/declspec?view=msvc-170, but `__declspec` itself is not allowed anywhere in concept definition. I'll remove `PAKM_Declspec` then.